tests/baselines/reference/taggedTemplateStringsWithOverloadResolution3_ES6.errors.txt

```diff
@@ -87,7 +87,7 @@ tests/cases/conformance/es6/templates/taggedTemplateStringsWithOverloadResolutio
     function fn5() { return undefined; }
     fn5 `${ (n) => n.toFixed() }`; // will error; 'n' should have type 'string'.
                      ~~~~~~~
-!!! error TS2339: Property 'toFixed' does not exist on type 'string'.
+!!! error TS2551: Property 'toFixed' does not exist on type 'string'. Did you mean 'fixed'?

```

Comment:
this looks weird until you [look up fixed](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/fixed) and find out that it's actually a deprecated member of String.